code/data/multi_country/convert-policies-raw-to-interim.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+import numpy as np
+import src.utils as cutil
+import json
+import operator
+import argparse
+
+
+def policy_conversion_parser():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--c",
+        choices=sorted(cutil.ISOS),
+        default=None,
+        type=str.upper,
+        help="Country ISO code",
+    )
+
+    parser.add_argument("--l", default=False, action="store_true", help="Print logs")
+
+    return parser
+
+
+parser = policy_conversion_parser()
+
+args = parser.parse_args()
+print_logs = args.l
+
+
+def get_country_list(country_input):
+    if country_input != None:
+        return [country_input]
+    return cutil.ISOS
+
+
+country_list = get_country_list(args.c)
+
+countries_wo_intensity = ["CHN", "IRN"]
+
+op_dict = {
+    ">": operator.gt,
+    "<": operator.lt,
+    ">=": operator.ge,
+    "<=": operator.le,
+    "=": operator.eq,
+}
+
+
+def apply_rule(df, src_policy, op_str, src_val, dst_rule, country_code):
+
+    dst_policy, dst_val = dst_rule
+
+    # Get operator function from operator string
+    op = op_dict[op_str]
+
+    if print_logs:
```

Comment:
no need to do this right now but sometime feel free to lookup the logging functionality in python. It's kind of a beast but can provide some useful functionality. ...but def not something we should do right now

Reply:
yep i know it's there, need to learn it!